server/routes/api.js

```diff
@@ -67,13 +68,21 @@ router.get('/messages', isLoggedIn, function(req, res, next) {
     initData.limit = parseInt(defaultLimit, 10);
   }
   if (pdwMode) {
-    var subquery = db.from('capcodes').where('ignore', '=', 0).select('id')
+    if (adminShow && req.isAuthenticated()) {
```

Comment:
Probably easier to move this check up to the pdwMode check 

Do 
`if (pdwMode && !req.isAuthenticated())` on line 70 

That way we aren't doubling up the same query as line 72 👍